applications/RomApplication/python_scripts/rom_nn_trainer.py

```diff
@@ -0,0 +1,260 @@
+import numpy as np
+import os
+import json
+
+import tensorflow as tf
+from keras.models import Model
+from keras import layers
+from keras.optimizers import AdamW
+from keras.callbacks import LearningRateScheduler
+
+import KratosMultiphysics
+
+
+class Rom_NN_trainer(object):
+
+    def __init__(self, general_rom_manager_parameters):
+
+        self.general_rom_manager_parameters = general_rom_manager_parameters
+        self.nn_parameters = self._SetNNParameters()
+
+    def _SetNNParameters(self):
+        defaults = self._GetDefaultNNParameters()
+
+        nn_params = self.general_rom_manager_parameters["NN"]
```

Comment:
```suggestion
        nn_params = self.general_rom_manager_parameters["neural_network"]
```
Use self-descriptive names in the input settings and avoid acronyms as much as possible.